README.md

```diff
@@ -78,7 +78,7 @@ public interface Publisher<T> {
 
 | ID                        | Rule                                                                                                   |
 | ------------------------- | ------------------------------------------------------------------------------------------------------ |
-| <a name="1.1">1</a>       | The number of `onNext` signaled by a `Publisher` to a `Subscriber` MUST NOT exceed the cumulative demand that has been signaled via that `Subscriber`’s `Subscription` |
+| <a name="1.1">1</a>       | The number of `onNext` signals sent by a `Publisher` to a `Subscriber` MUST always be less than or equal to the elements requested by that `Subscriber`´s `Subscription` |

```

Comment:
Yes, that’s better, we can improve it further by replacing “always” with “at all times” to put focus on the continued need to abide by this constraint.